README.md

```diff
@@ -16,35 +14,27 @@ And then execute:
 
     $ bundle
 
-### Usage
+## Pre-requisites
 
 Before starting this, you'll need to:
 
-- [read the documentation](https://docs.publishing.service.gov.uk/manual/ab-testing.html) on how to set up an a/b test. The cookie and header name in [govuk-cdn-config](https://github.com/alphagov/govuk-cdn-config/blob/master/ab_tests/ab_tests.yaml) must match the test name parameter that you pass to the Gem. The cookie name is case-sensitive.
-- configure Google Analytics (guidelines to follow)
+- [Read the documentation](https://docs.publishing.service.gov.uk/manual/ab-testing.html) for an overview on how a/b testing works on GOV.UK. 
+- The cookie and header name in [govuk-cdn-config](https://github.com/alphagov/govuk-cdn-config/blob/master/ab_tests/ab_tests.yaml) must match the test name parameter that you pass to the Gem. The cookie name is case-sensitive.
 
-To enable testing in the app, your Rails app needs:
+## Usage
 
-1. Some piece of logic to be A/B tested
-2. A HTML meta tag that will be used to measure the results, and which specifies
-the dimension to use in Google Analytics
-3. A response HTTP header that tells Fastly you're doing an A/B test
+### Outline 
 
-Start by defining which acceptance testing framework you will use. This gem
-supports both Capybara and ActiveSupport. In order to configure it, add this to
-your test helper file:
+To enable testing in the app, your Rails app needs:
 
-```
-GovukAbTesting.configure do |config|
-  config.acceptance_test_framework = :capybara # or :active_support
-end
-```
+1. [Some piece of logic to be A/B tested](#1-example-ab-test-logic)
```

Comment:
```suggestion
1. Some piece of logic to be A/B tested
```
I'd probably remove the links to the headers in this file. I assumed that it was an external link (like the one to the Tech docs). Alternative we should probably make no. 3 a link as well for consistency.